event/event_test.go

```diff
@@ -92,6 +92,26 @@ var newTests = []struct {
 			Data:               "test",
 		},
 	},
+	{
+		// valid CloudEvent with application/cloudevents+json Content-Type
```

Comment:
@alexdebrie should we accept CloudEvents with `application/json` content type?

Reply:
@mthenw Yea, I think we should. Not strictly according to CloudEvents spec, but 'be liberal in what you accept'.